### PR TITLE
Initialize Angular Modules

### PR DESCRIPTION
**Pull request by VladislavAntonyuk**

### CODE REVIEW

File: FunnyFridays/FunnyFridays.WebClient/webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+		new webpack.ContextReplacementPlugin(
+			/angular(\\|\/)core/,
+			path.resolve(__dirname, 'src'), // каталог с исходными файлами
+			{} // карта маршрутов
+		),
+		//позволяет оптимизировать код сборок
+		new webpack.optimize.CommonsChunkPlugin({
+			name: ['app', 'polyfills']
+		}),
+		//минифицирует код сборок
```
Comment (VitaliiMorenko) [inline FunnyFridays/FunnyFridays.WebClient/webpack.config.js]: Please, write code comments using english language

File: FunnyFridays/FunnyFridays.WebClient/Views/Home/Index.cshtml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+	}
+
+	function handleLocationError(browserHasGeolocation, infoWindow, pos) {
+		infoWindow.setPosition(pos);
+		infoWindow.setContent(browserHasGeolocation ?
+			'Error: The Geolocation service failed.' :
+			'Error: Your browser doesn\'t support geolocation.');
+		infoWindow.open(map);
+	}
+</script>
```
Comment (VitaliiMorenko) [inline FunnyFridays/FunnyFridays.WebClient/Views/Home/Index.cshtml:49]: I suggest to extract maps logic to separate angular component if it's possible

Comment (VladislavAntonyuk) [reply FunnyFridays/FunnyFridays.WebClient/Views/Home/Index.cshtml:49]: I have added scripts to maps.components.html but after renderer I saw only html code without scripts. I will investigate it.

File: FunnyFridays/FunnyFridays.WebClient/Views/Home/Index.cshtml [hunk 3]
```diff
@@ -0,0 +1,50 @@
+@{
+	ViewData["Title"] = "Home Page";
+}
+
+<app asp-prerender-module="ClientApp/dist/main-server">Loading...</app>
+
+@section scripts {
+	<script src="~/dist/vendor.js" asp-append-version="true"></script>
+	<script src="~/dist/main-client.js" asp-append-version="true"></script>
+}
```
Comment (matviygoff) [inline FunnyFridays/FunnyFridays.WebClient/Views/Home/Index.cshtml:10]: i think "map" functionality should be in separate view, not in Index

**Review by matviygoff — APPROVED** [review]

File: FunnyFridays/FunnyFridays.WebClient/Views/Home/Index.cshtml (hunk 3 above)
Comment (VladislavAntonyuk) [reply FunnyFridays/FunnyFridays.WebClient/Views/Home/Index.cshtml:10]: https://github.com/angular/angular/issues/4903

**Review by VitaliiMorenko — APPROVED** [review]

**Pull request merged**